src/chainstate/burn/db/sortdb.rs

```diff
@@ -684,7 +707,14 @@ impl <'a> SortitionHandleConn <'a> {
             }
         };
 
-        SortitionHandleConn::open_reader_stubbed(connection, &sn.burn_header_hash)
+        SortitionHandleConn::open_reader(connection, &sn.sortition_id)
+    }
+
+    fn get_pox_id(&self) -> Result<PoxId, db_error> {
```

Comment:
I think this method is duplicated below?  If so, can we DRY this up?

Reply:
One is just a wrapper around the other as a convenience method for calling `index_handle.get_pox_id()` without needing to instantiate a temporary index handle at every such call site.